content/resources/faq/_index.md

```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
 title at your choice. Then reroll portal with `gen3 kube-setup-portal`.
 
 
-**I updated Fence configuration but changes don't appear?**
+<h4>I updated Fence configuration but changes don't appear?</h4>
 
 After updating the Fence configuration, run `kubectl delete secret
 fence-config`, `gen3 kube-setup-fence` and roll Fence for configuration changes
 to take effect with `gen3 roll fence`.
 
 
-**GraphiQL on Query page form throwing errors.**
+<h4>GraphiQL on Query page form throwing errors.</h4>
 
```

Comment:
Is it "GraphiQL" or "GraphQL"?

Reply:
GraphiQL is the UI, GraphQL is the API :P

Reply:
In the Windmill it is GraphiQL